go.mod

```diff
@@ -87,7 +87,7 @@ require (
 	github.com/valyala/fastjson v1.6.4
 	github.com/vektah/gqlparser/v2 v2.5.16
 	github.com/xsleonard/go-merkle v1.1.0
-	go.uber.org/mock v0.4.0
+	go.uber.org/mock v0.4.1-0.20241003184824-d01ed3063816
```

Comment:
seems like they released v0.5.0 which has these changes https://github.com/uber-go/mock/releases/tag/v0.5.0
maybe we use that?